GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -118,6 +119,7 @@ GPURecoWorkflowSpec::GPURecoWorkflowSpec(GPURecoWorkflowSpec::CompletionPolicyDa
   mConfig.reset(new GPUO2InterfaceConfiguration);
   mConfParam.reset(new GPUSettingsO2);
   mTFSettings.reset(new GPUSettingsTF);
+  mNNClusterizerSettings.reset(new GPUSettingsProcessingNNclusterizer);
```

Comment:
You don't need this here, all these settings should already be made available from `mConfig->configProcessing.nn`